docs/docs-content/clusters/palette-virtual-clusters/configure-oidc-virtual-cluster.md

```diff
@@ -0,0 +1,106 @@
+---
+sidebar_label: "Configure OIDC for a Virtual Cluster"
+title: "Configure OIDC for a Virtual Cluster"
+description: "Learn how to configure OIDC for Palette Virtual Clusters."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 5
+tags: ["clusters", "cluster groups", "virtual clusters"]
+---
+
+
+Before deploying a virtual cluster, you must set up OpenID Connect (OIDC) Identity Provider (IDP) for the cluster group the virtual cluster belongs to. As virtual clusters are deployed to that cluster group, OIDC information in the cluster group configuration will be inserted into the virtual cluster configuration. 
+
+
+## Prerequisites
+
+- A healthy host cluster that you will use to create a cluster group.
```

Comment:
I think we can simplify this a just say, a cluster group and link to this [guide](https://docs.spectrocloud.com/clusters/cluster-groups/create-cluster-group).

Reply:
done.